assets/js/common/DateFilter/DateFilter.jsx

```diff
@@ -48,12 +55,58 @@ const parseInputOptions = (options) =>
 
 const getSelectedOption = (options, value) => {
   const selectedId = Array.isArray(value) ? value[0] : value;
+  if (selectedId === 'custom') {
+    return ['custom', value[1], () => toHumanDate(value[1])];
+  }
   if (typeof selectedId === 'string') {
     return options.find((option) => option[0] === selectedId);
   }
   return undefined;
 };
 
+function Tick() {
+  return (
+    <span className="absolute inset-y-0 right-0 flex items-center pr-4">
+      <svg
```

Comment:
**suggestion:** we might use `EOS_CHECK` icon from eos icon-set.

Reply:
Good point. I'd rather do it in another PR, as we might also refactor the `Filter` component. I prepared it already: https://github.com/trento-project/web/pull/2870